Terminal.Gui/Core/PosDim.cs

```diff
@@ -165,7 +165,7 @@ public static Pos Center ()
 			return pCenter;
 		}
 
-		class PosAbsolute : Pos {
+		internal class PosAbsolute : Pos {
```

Comment:
Changing `PosAbsolute` to be a 'psuedo-public' by exosing it as `internal` like this gives me angst. 

It is a slippery slope towards taking @migueldeicaza's brilliant (but confusing) design of 'PosDim` to the next level of complexity (and confusion). Right now, the public API for 'PosDim' is very constrained and none of the internals are used anywhere outside of `View` (except the unit tests). By doing this, I think you start to break this.

I have not had the time to fully study the issue represented by this bug/PR, so I can't offer any guidance on if there's a different way to solve this problem without exposing what is really an internal implementation detail (`PosAbsolute`) to other parts of the framework. 

If you say "nope, this is the only way to fix it" then I'll approve it and we can move on. But I'd like to ask you to consider what I'm asking. Thanks!

Reply:
@tig I really am considering what you are asking for but I really need it as I also needed the `internal class DimAbsolute` which is already `internal`. If not, there is no way I know how to differentiate the type. Thanks.

Reply:
Ok, let's do it. 

But would you do me the favor of creating an Issue to track whether these should be `public` or not? That will force me to continue to think about it, and the implications.